docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -92,6 +92,13 @@ Select `yes` to delete all monitors associated with the project ID being pushed.
 it using a _different_ ID. Select `yes` to create duplicates of all monitors in the project.
 ====
 
+[NOTE]
+====
+If the journey code in a project uses an external Node.js package,
+that package will be bundled with the journey code on `push`.
+However, you can't `push` bundles that are more than 800 Kilobytes.
+====
```

Comment:
I wasn't sure if I should also mention that "External packages that use native bindings will not work" in the command reference. Will it prevent the user from pushing the project or can they push it, but it won't work as expected?